Userland/Libraries/LibWeb/HTML/History.cpp

```diff
@@ -44,14 +44,68 @@ WebIDL::ExceptionOr<void> History::replace_state(JS::Value data, String const&,
     return shared_history_push_replace_state(data, url, IsPush::No);
 }
 
+// https://html.spec.whatwg.org/multipage/history.html#dom-history-length
+WebIDL::ExceptionOr<u64> History::length() const
+{
+    // 1. Let document be history's associated Document. (NOTE: Not necessary)
```

Comment:
Still wrong, the steps from https://html.spec.whatwg.org/multipage/history.html#dom-history-length are:

```
1. If this's associated Document is not fully active, then throw a "SecurityError" DOMException.
2. Return the number of entries in the top-level browsing context's joint session history.
```

Reply:
OK